add_package.yml

```diff
@@ -27,3 +27,9 @@
       shell: "gem2rpm -t {{ rubygem_spec_template }} {{ gem_name }}-{{ version }}.gem > {{ inventory_hostname }}.spec"
       args:
         chdir: "packages/{{ inventory_hostname }}"
+
+    - name: 'Create git branch'
+      command: "git checkout -b {{ gem_name }}-version"
```

Comment:
was that intended to be `{{ gem_name }}-{{ version }}`?

Reply:
It was! Good catch